cardano-cli/src/Cardano/CLI/Byron/Commands.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
Which standard? This just switches from British to American spelling.

Reply:
Stylish Haskell crashes on the British spelling, which was introduced in `8.2`

Reply:
```
Language.Haskell.Stylish.Parse.parseModule: could not parse ./cardano-config/src/Cardano/TracingOrphanInstances/Shelley.hs: ParseFailed (SrcLoc "<unknown>.hs" 159 18) "GeneralizedNewtypeDeriving language extension is not enabled. Please add {-# LANGUAGE GeneralizedNewtypeDeriving #-} pragma at the top of your module."
```

Reply:
I've raised a PR against `stylish-haskell`:

https://github.com/jaspervdj/stylish-haskell/issues/295

Reply:
Clearly the British spelling is the correct one :wink: